src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeption.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User validUser;
+    private static final String EXCEPTION_MESSAGE =
+            "RegistrationException should be thrown in this case.";
+
+    @BeforeAll
+    public static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        validUser = new User("goodlog", "password", 18);
+    }
+
+    @BeforeEach
+    public void preparing() {
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_validUser_Ok() {
```

Comment:
please fix according the next pattern (be attentive about small and capital letters)
```suggestion
    public void register_validUser_ok() {
```